### DISCUSSION TITLE
Full screen settings menu

### DISCUSSION THREAD

**Discussion by smileyhead** [ppy/osu#12990] [Ideas]
First of all, I'm not a developer, so I can't do this myself, but I have some experience in designing UIs for others, so I know what I'm doing. Also, this seems to be an easy fix.

I think the settings menu should be full screen and aligned to the centre. This is why:
- It would allow for the menu to be wider: I've noticed in osu!stable that some languages have considerably longer strings than in English, making the UI look ugly and making sliders shorter (this is especially an issue with the beatmap difficulty hide sliders and their precision is already less than optimal in English, let alone with longer strings).
- It would look nicer: a full screen menu (with the transparent background kept, of course) with the text being centred looks way better than a thin panel on the side.

Here's a concept. The text has about twice as much space with this design as with the original design.

![image](https://user-images.githubusercontent.com/15932372/43387937-1f46d446-93e8-11e8-8770-21e2c09eab50.png)

**Comment by peppy** [comment]
Maybe as an optional toggle. They are designed to take up a small portion of the screen as settings apply in real-time, and it is useful to get visual feedback of the changes they make to the UI/gameplay.
